### ISSUE TITLE
popup: remove wrapper?

### ISSUE THREAD

**Issue by wkeese** [ibm-js/delite#299]
As mentioned in #266, consider removing popup's wrapper node completely, and just position the popup itself.

That would also resolve a weirdness where both the popup and popup wrapper have either `aria-label` or `aria-labelledby` defined, which seems redundant.   It also (currently) doesn't work well when the popup defines `aria-labelledby` instead of `aria-label`: the popup wrapper still defines `aria-label`.

Note: Currently we need to keep the wrapper to support IE11, due to flexbox bugs when the popup is display:flex.    The workaround is to encase the popup in another display:flex node.
